### ISSUE TITLE
Plugin settings implementation

### ISSUE THREAD

**Issue by Tomkov1c** [Flow-Launcher/Flow.Launcher.PluginsManifest#391]
I've been trying to implement settings into my c# plugin for weeks. I looked at other plugins who have them and copy pasted their code. No luck. Xaml elements don't load. Please help

**Comment by taooceros** [comment]
Have you implemented this interface? https://www.flowlauncher.com/docs/#/API-Reference/Flow.Launcher.Plugin/ISettingProvider

**Comment by Tomkov1c** [comment]
Yes. Made a UserControl page and everything, It doesn't show up.

**Comment by Tomkov1c** [comment]
https://github.com/Tomkov1c/Lorem-Ipsum-Generator

**Comment by Tomkov1c** [comment]
I tried copying another plugin's settings implementation. Didn't work. Pls help
